packages/core/src/browser/keyboard/keys.ts

```diff
@@ -344,9 +344,9 @@ export namespace KeyCode {
      * `keyIdentifier` is used to access this deprecated field:
      * https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier
      */
-    export function toKey(event: KeyboardEvent): Key {
+    export function toKey(event: KeyboardEvent, dispatch: 'code' | 'keyCode'): Key {
```

Comment:
The following would be a breaking change, if it cannot be avoided (using optional parameters), we should add an entry in the `changelog` to reflect the breakage.

Reply:
I was able to just make the parameter optional.